packages/ethereum/web3-ethereum/src/index.ts

```diff
@@ -392,15 +357,15 @@ implements EthereumProvider.EthereumFunctionCall {
 	}
 
 	private getGasOptions(options: EthereumProvider.EthereumSendOptions) {
-		const gasOptions: InternalGasOptions = {
+		const gasOptions: Web3EthereumGasOptions = {
 			// If we won't setup null here explicitly sometimes MetaMask
 			// interpret this option as 'suggested by website' in their UI
 			// @see https://stackoverflow.com/questions/68926306/how-to-avoid-this-gas-fee-has-been-suggested-by-message-in-metamask-using-web3
-			gasPrice: null as any,
+			gasPrice: null as unknown as Web3EthereumGasOptions["gasPrice"],
```

Comment:
You can remove this null. And moreover this is major issue in case when user is on v1.9.0+ (I've checked and already fixed it on master branch). After that we're safe to use always web3 ^1.0.0  in dependencies